.github/labeler.yml

```diff
@@ -32,7 +32,7 @@ tests:
       - all-globs-to-all-files: ['!tests/requirements.txt', '!perf-tests/requirements.txt']
 
 documentation:
-  - head-branch: ['^docs/', '^doc/']
+  - head-branch: ['^site/']
```

Comment:
@ADubhlaoich I'm re-thinking this as I've taken a second look.  This actually means branches named `docs/mybranch` which I think we want to keep.  Apologies for the mistake.